README.md

```diff
@@ -212,10 +212,29 @@ console.log(
 ```
 
 
-## CommonJS modules
+## Others
+
+### Write code in CommonJS modules
 
 If you want to use CommonJS modules, `require` the code from `@puppeteer/replay/cjs`. For example,
 
 ```js
 const { stringify } = require('@puppeteer/replay/cjs');
 ```
+
+### Create a Chrome extension for Recorder
+
+You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extending DevTools.
+
+This repository contains an [example extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension). Once installed, the Recorder will have a new export option **Export as a Custom JSON script** in the [export dropdown](https://developer.chrome.com/docs/devtools/recorder/#export-flows).
+
+To load the example into Chrome DevTools. Follow these steps:-
+
+1. Download the [chrome-extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension) folder.
+2. [Load the folder as unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked) in Chrome. 
+3. [Open a recording](https://developer.chrome.com/docs/devtools/recorder/#record) in the Recorder.
+4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). A new **Export as a Custom JSON script** menu is added.
+
+Click and watch the video demo below:
+
+[![Watch the video demo](https://user-images.githubusercontent.com/5917927/172872574-15ad8bea-142a-4972-bf1d-bf1379a955ba.png)](https://youtu.be/TCxIfbxgypQ)
```

Comment:
nit: as far as I understand, this `[]` part is added as an `alt` tag to the image. So, we can try to be descriptive there instead of action oriented. We can say something like `Demo video that shows how to extend export options in Recorder panel by adding a Chrome extension`. WDYT?